<launch>
	<node name = "controller_node"
    		pkg = "ros_robotics"
    		type = "controller_train5_crab2"
    		output = "screen"/>


	<!--node name = "terrain_model_node"
    		pkg = "ros_robotics"
    		type = "terrain_model_train5"
    		/-->

	<!--node name = "motor_model_node"
    		pkg = "ros_robotics"
    		type = "motor_model_train5"
    		/>

	<node name = "motor_model_service_node"
    		pkg = "ros_robotics"
    		type = "motor_model_service_train5"
    		/>

	<node name = "Xt_handler_node"
    		pkg = "ros_robotics"
    		type = "Xt_train5.py"
    		/-->



</launch>
